### PR TITLE
[ISV-1504] Integration testing of the Release pipeline for E2E Testing.

### PR DESCRIPTION
**Pull request by haripate**

### CODE REVIEW

File: .github/workflows/e2e.yml [hunk 1]
```diff
@@ -10,9 +10,12 @@ on:  # yamllint disable-line rule:truthy
       - Build
     branches:
       - main
+      - ISV-1504
```
Comment (wcheang) [inline .github/workflows/e2e.yml:13]: I'm assuming this is just for testing, and will be removed before merging, right?

File: .github/workflows/e2e.yml [hunk 2]
```diff
@@ -153,7 +156,6 @@ jobs:
             --param bundle_path=operators/test-e2e-operator/0.0.7-$SHORT_SHA \
             --param env=stage \
             --param submit=false \
-            --param pipeline_image=quay.io/redhat-isv/operator-pipelines-images:${{ github.sha }} \
```
Comment (wcheang) [inline .github/workflows/e2e.yml:156]: Why is this being removed? Wouldn't that mean the pipeline_image will use the default, which is the released image, and not what's on the PR?

File: .github/workflows/e2e.yml [hunk 3]
```diff
@@ -10,9 +10,12 @@ on:  # yamllint disable-line rule:truthy
       - Build
     branches:
       - main
+      - ISV-1504
     types:
       - completed
   workflow_dispatch:
+  pull_request:
```
Comment (Allda) [inline .github/workflows/e2e.yml:17]: This should also be removed before a merge.

File: .github/workflows/e2e.yml [hunk 4]
```diff
@@ -185,7 +187,6 @@ jobs:
             --param env=stage \
             --param preflight_min_version=0.0.0 \
             --param ci_min_version=0.0.0 \
-            --param pipeline_image=quay.io/redhat-isv/operator-pipelines-images:${{ github.sha }} \
```
Comment (Allda) [inline .github/workflows/e2e.yml:188]: This should also not be removed.

File: .github/workflows/e2e.yml [hunk 5]
```diff
@@ -201,6 +202,32 @@ jobs:
...
+            --workspace name=image-data,volumeClaimTemplateFile=templates/workspace-template-small.yml \
+            --workspace name=registry-credentials,secret=registry-dockerconfig-secret \
+            --showlog
+
+      - name: Check if Release pipeline passed
+        run: |
+          PIPELINERUN_NAME=$(oc get pr --no-headers -l tekton.dev/pipeline=operator-release-pipeline -o custom-columns=":metadata.name")
+          PIPELINERUN_SUCCEEDED=$(oc get pr $PIPELINERUN_NAME -o jsonpath={'.status.conditions[].status'})
+          if [[ "$PIPELINERUN_SUCCEEDED" != "True" ]]; then
+            oc get pr
```
Comment (Allda) [inline .github/workflows/e2e.yml:227]: what does `pr` represent?

Comment (Allda) [reply .github/workflows/e2e.yml:227]: nevermind... it is pipeline run

File: .github/workflows/e2e.yml (hunk 1 above)
Comment (haripate) [reply .github/workflows/e2e.yml:13]: Yes, it is only for testing purposes. I will remove the config before merge.

**Review by amisstea — APPROVED** [review]

**Comment by amisstea** [comment]
@haripate please squash all the commits when merging this. Ty!

**Pull request merged**